test/testcafe/framework/page-objects/EnrollSecurityQuestionPageObject.js

```diff
@@ -10,14 +11,25 @@ export default class EnrollSecurityQuestionPageObject extends BasePageObject {
   }
 
   clickChooseSecurityQuestion() {
+    if (userVariables.v3) {
+      return this.form.selectRadioButtonOptionByValue('','predefined');
+    }
     return this.form.selectRadioButtonOption('sub_schema_local_credentials', 0);
   }
 
   clickCreateYouOwnQuestion() {
+    if (userVariables.v3) {
+      return this.form.selectRadioButtonOptionByValue('', 'custom');
+    }
     return this.form.selectRadioButtonOption('sub_schema_local_credentials', 1);
   }
 
   selectSecurityQuestion(index) {
+    // In v3 widget, "Choose a security question" is also a value in dropdown
+    // We need to skip that value, by increasing the index by 1
+    if (userVariables.v3) {
+      index = index + 1;
+    }
```

Comment:
I am a little confused by this, does v2 not also have the same options in the security question select box?